doc/models/text_classification.rst

```diff
@@ -9,7 +9,7 @@ Input: Text sequence
 Output: Class label
 
 
-tree_lstm_model
+TreeLstmModel
```

Comment:
based on your changes https://github.com/allenai/deep_qa/pull/159/files#diff-a095eea3e0a97386695f6b99083d0bbbR22, maybe keep it consistent and have this be `TreeLSTMModel`?